networks/rococo.toml

```diff
@@ -0,0 +1,23 @@
+[settings]
+timeout = 1000
+node_spawn_timeout = 300
+
+[relaychain]
+chain = "rococo-local"
+default_command = "./polkadot"
+default_args = ["--insecure-validator-i-know-what-i-do"]
```

Comment:
This flag is already added by zombienet and will result in an error when added twice. 
```suggestion
```